single-node-k8s.sh

```diff
@@ -67,7 +67,7 @@ function set_env() {
     export MEMORY="8192"
     export CEPH_CSI_RUN_ALL_TESTS=true
     # downloading rook images is sometimes slow, extend timeout to 15 minutes
-    export ROOK_VERSION='v1.3.8'
+    export ROOK_VERSION='v1.3.9'
```

Comment:
use v1.4.0 which is released yesterday?

Reply:
Not sure if that works flawlessly or if we need modifications somewhere. This is a minor update, so it should not give any regressions.

Reply:
In 1.4.0 I don't see any other modifications will send a PR for that